iphone/Maps/Classes/CarPlay/CarPlayService.swift

```diff
@@ -374,6 +374,13 @@ extension CarPlayService: CPMapTemplateDelegate {
     isUserPanMap = true
   }
 
+  func mapTemplate(_ mapTemplate: CPMapTemplate, didUpdatePanGestureWithTranslation translation: CGPoint, velocity: CGPoint) {
+    let displayScale = self.window?.traitCollection.displayScale ?? 1
+    let scaledTranslation: CGPoint = translation
+      .applying(CGAffineTransform(scaleX: -1*displayScale, y: -1*displayScale))
```

Comment:
1. Why is this transformation needed?
2. Will `-displayScale` work?
3. https://developer.apple.com/documentation/uikit/uiview/1622657-contentscalefactor

Reply:
Thanks for your feedback!

> Why is this transformation needed?

I have to convert the points from `translation: CGPoint` to pixels (see [here](https://www.paintcodeapp.com/news/ultimate-guide-to-iphone-resolutions)) by multiplying them with the contentScaleFactor. I changed the code to make this more obvious.

> Will -displayScale work?

Sure it will and looks much nicer. Thanks!

> https://developer.apple.com/documentation/uikit/uiview/1622657-contentscalefactor

Thanks that helped to understand the difference between points and pixels!